js/try.js

```diff
@@ -261,33 +353,34 @@ const fetchAndParseTimetable = async (url) => {
       )
       .join(', ')
 
+    const startHour = extractTime(info[7].text);
+    const endHour = extractTime(info[8].text);
+
+    const day = new Date(date);
+    day.setDate(day.getDate() + dayIndex);
+
+    const startDate = new Date(day);
```

Comment:
Can use a single `date` field, which is converted into 2 dates later before conversion

Reply:
what if a lecture is more than an hour long? then you still need to store the length. i think this is the best way to represent the start and end of a lecture

Reply:
I meant that you don't need to store the same date twice, you can create these later when it is needed for the ICS library